lib/progressTracker.js

```diff
@@ -18,6 +18,7 @@ const defaults = {
   renderResultsTable: true,
   renderLatencyTable: false
 }
+const spinner = ora('running...')
```

Comment:
Can you please instantiate this on first use instead?

Reply:
that will reintroduce the problem again :)

Reply:
Keeping it global and instantiating it only on first use should be identical to this.

Reply:
it is not, see the change I did which delays initialization and see the test I added failing. _when_ it is initialized is what makes the difference and that's also part of the reason why I couldn't quite figure out the root cause

Reply:
ok no wait a second, there's something gnarly at play here. the test is failing with this version of the code on my machine, but it is not failing on CI. looking into that

Reply:
ok the reason was unrelated. the test file for the progress tracking where I added this test was wrongly named and its tests were never run (I was running that file directly when developing locally, hence I was I saw it failing)